app/assets/stylesheets/application.css.scss

```diff
@@ -63,6 +63,23 @@ footer {
   padding: 10px;
 }
 
+.splash-title {
+  font-family: 'Miltonian Tattoo', cursive;
+  margin: 0;
+  text-align: center;
+}
+
+.splash-body {
+  font-family: 'Miltonian Tattoo', cursive;
+  text-align: center;
+  a {
+    color: black;
+  }
+  a:hover {
+    color: darken(blue, 30%)
```

Comment:
I deleted Hound's comment, but you do indeed need a semicolon at the end of this line